microsoft-edge/devtools-guide-chromium/overview.md

```diff
@@ -502,94 +540,98 @@ In addition to Panel tools and Drawer tools, DevTools includes the following too
 
 DevTools provides lots of features and functionality to use with your website.  You can access the different parts of DevTools in many ways, but often a fast way is to use the Command Menu.
 
-![the Command Menu in DevTools](./overview-images/devtools-intro-command-menu.png)
+![the Command Menu in DevTools](./overview-images/command-menu.png)
 
 In the Command Menu, the tools are called "panels"; for example, the **Elements** tool is called the **Elements** panel.  To switch to the **Elements** tool, you select the **Elements** tab.
 
-To open the Command Menu, do either of the following:
+1. To open the Command Menu, do either of the following:
 
-*  Click the **Customize and control DevTools** (![Customize and control DevTools](./overview-images/customize-and-control-devtools-icon-light-mode.png)) button, and then select **Run command**.
+   *  Click the **Customize and control DevTools** (![Customize and control DevTools](./overview-images/customize-and-control-devtools-icon-light-mode.png)) button, and then select **Run command**.
+   *  Press **Ctrl+Shift+P** (Windows, Linux) or **Command+Shift+P** (macOS).
 
-*  Press **Ctrl+Shift+P** (Windows, Linux) or **Command+Shift+P** (macOS).
+   The Command Menu allows you to type commands to display, hide, or run features in DevTools.
 
+1. In the Command Menu, start typing the name of a tool, such as **changes**, and then select a **Show** command, such as **Show Changes**.  The Command Menu displays the matching commands:
 
-The Command Menu allows you to type commands to display, hide, or run features in DevTools.  With the Command Menu open, enter the word **changes**, and then select **Drawer: Show Changes**.
+   ![Command menu displays the options after you type 'cha', including 'Show Changes tool in the Drawer'](./overview-images/command-menu-show-changes.png)
```

Comment:
Global in .md: Find "Drawer", change carefully to "Quick View", while checking rendered article for context.  @captainbrosset

Reply:
Good find, I guess this one slipped through the cracks! I'll push a commit here.